lib/documents/schemas/aaib_reports.json

```diff
@@ -1,4 +1,5 @@
 {
+  "target_stack": "live",
```

Comment:
To make it easier to review, I'd move the config update into its own commit (i.e. add all the `target_stack` properties in one commit, then the logic that uses it would be added in the next commit).